### ISSUE TITLE
[Feature Request] - Support server timezones other than UTC

### ISSUE THREAD

**Issue by nx10** [veeso/suppaftp#26]
## Description

[SuppaFtp::FtpStream::mdtm](https://docs.rs/suppaftp/latest/suppaftp/struct.FtpStream.html#method.mdtm) assumes the server timezone is UTC. I used this hack to change it to a different time zone afterwards:

```Rust
fn fix_timezone(datetime_wrong_tz: &DateTime<Utc>, actual_tz: Tz) -> chrono::LocalResult<DateTime<Tz>> {
    datetime_wrong_tz.naive_utc().and_local_timezone(actual_tz)
}
```

But I think there should be a way to either directly return a `chrono::NativeDateTime`, a second method that does so, or a way to provide server time zone.

**Comment by veeso** [comment]
Thanks for reporting this issue.

I've decided the best option is to return `NaiveDateTime`, since there's no way to know the server timezone. The user can then apply a certain Timezone if they know what it is.

Fixed in 4.6.0

**Comment by nx10** [comment]
Fantastic! Thank you for the quick response.
